packages/common/src/api/developer-apps.ts

```diff
@@ -0,0 +1,80 @@
+import { z } from 'zod'
+
+import { ID } from 'models/Identifiers'
+import { createApi } from 'src/audius-query'
+import { encodeHashId } from 'utils/hashIds'
+
+const descriptionMaxLength = 28
+
+export const developerAppSchema = z.object({
+  userId: z.number(),
+  name: z.string().min(5),
```

Comment:
Not sure if name needs a min,  maybe run by Forrest?

Reply:
word good point, ill remove the min anyway